target/linux/sunxi/config-5.10

```diff
@@ -430,6 +434,7 @@ CONFIG_SUN8I_DE2_CCU=y
 CONFIG_SUN8I_H3_CCU=y
 CONFIG_SUN8I_R40_CCU=y
 CONFIG_SUN8I_R_CCU=y
+# CONFIG_SUN8I_THERMAL is not set
```

Comment:
I think we can enable this at least for a7 and a53, what do you think?
In case of crypto drivers, we may enable them by adding kernel modules later.